tests/test_pagination.py

```diff
@@ -18,13 +18,17 @@ def name(self):
 
 
     def test_run(self):
+        # skip 'locations' stream as there is not much info about
+        # limit of records returned in 1 page
+        # Documentation: https://help.shopify.com/en/manual/locations/setting-up-your-locations
+        excepted_streams = {'locations'}
         with self.subTest(store="store_1"):
             conn_id = self.create_connection(original_credentials=True)
-            self.pagination_test(conn_id, self.store_1_streams)
+            self.pagination_test(conn_id, self.store_1_streams - excepted_streams)
 
         with self.subTest(store="store_2"):
             conn_id = self.create_connection(original_properties=False, original_credentials=False)
-            self.pagination_test(conn_id, self.store_2_streams)
+            self.pagination_test(conn_id, self.store_2_streams - excepted_streams)
```

Comment:
For this test could you please add the assertions that verify records are unique across pages by checking primary key values. It should leverage tuples in case of any compound primary keys and the assertCountEqual method. https://github.com/singer-io/tap-hubspot/blob/290e5c051839b01e43739d25e1c46b6ddef5a749/tests/test_hubspot_pagination_test.py#L118-L127

Reply:
Done the changes.